index.html

```diff
@@ -55,5 +59,236 @@ <h4 class="headline-sm-suport">LET'S CONNECT</h4>
     </section>
     <!--headline section ends-->
 
+    <!--Card works starts-->
+    <section class="works">
+        <!-- Card 3 starts -->
+        <li class="works-card">
```

Comment:
- [ ] This `li` tag doesn't make sense here as it is not wrapped by any ul. what yo can do is take a ul at line 64 and then take each works card as an individual `li`. that's how it will be semantically correct. kindly consider restructuring the cards the way I just mentioned.